subsys/bluetooth/shell/bt.c

```diff
@@ -526,6 +528,36 @@ static int cmd_init(int argc, char *argv[])
 	return 0;
 }
 
+static int cmd_hci_cmd(int argc, char *argv[])
+{
+	u16_t op;
+	struct net_buf *buf = NULL;
+	int err;
+
+	if (argc < 2) {
+		return -EINVAL;
+	}
+
+	op = strtoul(argv[1], NULL, 16);
+
+	if (argc > 2) {
+		int i;
+
+		buf = bt_hci_cmd_create(op, argc - 2);
+
+		for (i = 2; i < argc; i++) {
+			net_buf_add_u8(buf, strtoul(argv[i], NULL, 16));
+		}
+	}
+
+	err = bt_hci_cmd_send_sync(op, buf, NULL);
```

Comment:
What about command complete with return parameters, printing them will be nice.

Reply:
Agreed, it'd be nice to be able to see those

Reply:
I guess we could decode de status since the commands are normally not really known, that said perhaps we print the hexdump?

Reply:
I was assuming you'd just hexdump the bytes, don't think it needs to be any fancier than that. Same as with the command parameters.